commands.go

```diff
@@ -1249,6 +1256,139 @@ func (c *cmdable) SUnionStore(destination string, keys ...string) *IntCmd {
 
 //------------------------------------------------------------------------------
 
+// X represents stream message response.
+type X struct {
+	Key  string
+	Messages []XMessage
+}
+
+// XStreamCheckpoint represents a query into a stream
+type XStreamCheckpoint struct {
+	Key string
+	Checkpoint string
+}
+
+// XMessage represents messages within the stream response
+type XMessage struct {
+	ID string
+	Values map[string]interface{}
```

Comment:
It looks inconsistent that we add `els ...interface{}`, but retrieve `Values map[string]interface{}`. If order of the message fields does not matter I would prefer to have `values map[string]interface{}` everywhere.